libpod/container_internal_linux.go

```diff
@@ -2495,9 +2495,6 @@ func (c *Container) getOCICgroupPath() (string, error) {
 		}
 		return c.config.CgroupParent, nil
 	case c.config.CgroupsMode == cgroupSplit:
-		if c.config.CgroupParent != "" {
```

Comment:
for this, if the container is joining a pod and `usePodCgroup` is specified, would that override this here or cause a conflict?

Reply:
it would override and ignore `UsePodCgroup` in the same way as setting a `cgroup-parent` does